examples/kotlin/BUILD.bazel

```diff
@@ -21,28 +22,41 @@ kt_js_library(
     deps = [":kotlinx-html-js"],
 )
 
+# Copy boostrap.js to the bin folder as _bootstrap.js
+# so that relative import to `./hello.js` is valid
+genrule(
+    name = "bootstrap",
+    srcs = ["bootstrap.js"],
+    outs = ["_bootstrap.js"],
+    cmd = "cp $< $@",
+)
+
 rollup_bundle(
     name = "bundle",
     srcs = ["hello.js"],
-    entry_point = "bootstrap.js",
+    config_file = "rollup.config.js",
+    entry_point = "_bootstrap.js",
+    # TODO: make this example work with format = "esm"
+    format = "cjs",
+    output_dir = True,
     deps = [
         "@npm//kotlin",
         "@npm//kotlinx-html-js",
     ],
 )
 
+terser_minified(
+    name = "bundle.min",
+    src = ":bundle",
+)
+
 web_package(
     name = "package",
-    assets = [
-        # For differential loading, we supply both an ESModule entry point and an es5 entry point
-        # The injector will put two complimentary script tags in the index.html
-        ":bundle.min.js",
-        ":bundle.min.es2015.js",
-    ],
-    data = [":bundle"],
+    data = [":bundle.min"],
     index_html = "index.html",
 )
 
+# TODO: fix prodserver; require.js is needed if rollup_bundle output is cjs
```

Comment:
can remove since we have only esm

Reply:
actually to get this example working I had to use ```    # TODO: make this example work with format = "esm"
    format = "cjs",
``` in the rollup_bundle to match what the old rollup_bundle produced with its incorrect `format = "es"` bug

Reply:
node takes issue with `import('./hello.js')` since it doesn't know that its an esm module and it imports it as CJS

Reply:
the prodserver never worked in the old example either